debian/changelog

```diff
@@ -1,3 +1,11 @@
+containerd.io (1.7.28-1) release; urgency=medium
+
+  * Update containerd binary to v1.7.28
+  * Update runc binary to v1.3.0
+  * Update Golang runtime to 1.23.12
```

Comment:
@vvoland if we do this release, we need to override the Go version in the build pipeline, because containerd upstream did a release just before 1.23.12 was released, so it's still pointing to 1.23.11 as default;
https://github.com/containerd/containerd/blob/v1.7.28/contrib/Dockerfile.test#L32